src/main/kotlin/no/nav/syfo/dialogmotekandidat/database/DialogmotekandidatStoppunktQuery.kt

```diff
@@ -61,6 +63,37 @@ fun DatabaseInterface.getDialogmotekandidatStoppunktList(
         }
     }
 
+const val queryUpdateDialogmotekandidatStoppunktStatus =
+    """
+        UPDATE DIALOGMOTEKANDIDAT_STOPPPUNKT SET status=?, processed_at=? WHERE uuid = ?
```

Comment:
her er det en p for mye i stoppunkt

Reply:
Her har vi visst brukt tre P-er i tabellnavnet i en tidligere PR 🤦‍♂️ Får evt migrere det i en egen PR tenker jeg.